seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json

```diff
@@ -644,40 +644,46 @@
       "type": "bonfire",
       "title": "Problem 18: Maximum path sum I",
       "tests": [
-        "assert.strictEqual(euler18(), 1074, 'message: <code>euler18()</code> should return 1074.');"
+        "assert.strictEqual(maximumPathSumI(testTriangle), 23, 'message: <code>maximumPathSumI(testTriangle)</code> should return 23.');",
+        "assert.strictEqual(maximumPathSumI(numTriangle), 1074, 'message: <code>maximumPathSumI(numTriangle)</code> should return 1074.');"
       ],
-      "solutions": [],
+      "solutions": ["const testTriangle = [[3, 0, 0, 0],\n  [7, 4, 0, 0],\n  [2, 4, 6, 0],\n  [8, 5, 9, 3]];\n\nconst numTriangle = [[75, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n  [95, 64, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n  [17, 47, 82, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n  [18, 35, 87, 10, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n  [20, 4, 82, 47, 65, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n  [19, 1, 23, 75, 3, 34, 0, 0, 0, 0, 0, 0, 0, 0, 0],\n  [88, 2, 77, 73, 7, 63, 67, 0, 0, 0, 0, 0, 0, 0, 0],\n  [99, 65, 4, 28, 6, 16, 70, 92, 0, 0, 0, 0, 0, 0, 0],\n  [41, 41, 26, 56, 83, 40, 80, 70, 33, 0, 0, 0, 0, 0, 0],\n  [41, 48, 72, 33, 47, 32, 37, 16, 94, 29, 0, 0, 0, 0, 0],\n  [53, 71, 44, 65, 25, 43, 91, 52, 97, 51, 14, 0, 0, 0, 0],\n  [70, 11, 33, 28, 77, 73, 17, 78, 39, 68, 17, 57, 0, 0, 0],\n  [91, 71, 52, 38, 17, 14, 91, 43, 58, 50, 27, 29, 48, 0, 0],\n  [63, 66, 4, 68, 89, 53, 67, 30, 73, 16, 69, 87, 40, 31, 0],\n  [4, 62, 98, 27, 23, 9, 70, 98, 73, 93, 38, 53, 60, 4, 23]];\nfunction maximumPathSumI(triangle) {\n  let maxSum = triangle.slice();\n\n  for (let i = triangle.length - 1; i > 0; i--) {\n    let currentRow = maxSum[i];\n    let previousRow = maxSum[i - 1];\n    const temp = [];\n    for (let j = 0; j < i; j++) {\n      temp.push(Math.max((currentRow[j] + previousRow[j]), (currentRow[j + 1] + previousRow[j])));\n    }\n    maxSum[i - 1] = temp;\n    maxSum.pop();\n  }\n  return maxSum[0][0];\n}"],
       "translations": {},
       "challengeSeed": [
-        "function euler18() {",
+        "function maximumPathSumI(triangle) {",
         "  // Good luck!",
         "  return true;",
         "}",
         "",
-        "euler18();"
+        "const testTriangle = [[3, 0, 0, 0],",
+        "                      [7, 4, 0, 0],",
+        "                      [2, 4, 6, 0],",
+        "                      [8, 5, 9, 3]];",
+        "",
+        "const numTriangle = [[75, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],",
```

Comment:
Thanks for your patience @scissorsneedfoodtoo, can you move `numTriangle` to the head please, just like before in other challenges.

Many thanks :+1:

Reply:
@Bouncey, no problem. I can do that now.